### ISSUE TITLE
[Autonomy] Integration of Route Planning Software

### ISSUE THREAD

**Issue by SwapnilPande** [vanderbiltrobotics/NASA-RMC-2018-2019#8]
**Which section of robot code is this for?**
Route Planning and Execution

**Description of feature**
* Find suitable software package to efficiently generate route between two waypoints provided by high level controller
* Build ROS node for package or integrate existing ROS package into our ROS project

**Comment by AmyChen-Qingxin** [comment]
What does this assignment mean?

**Comment by BaibhaVatsa** [comment]
We have been more focused lately on the pure pursuit so have not made any headway in this. ROS environment and everything has been set up on everyone's laptops.

**Comment by partlygloudy** [comment]
Yeah don't worry about this too much for now. Since you guys got a version of A* working already, pure pursuit should be the focus at the moment.
